cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js

```diff
@@ -78,15 +80,30 @@ export default class ColumnActionsDropdown extends Component {
     - 2: Works only when there are 2 selected columns.
 
     */
-    this.directives = [
+    this.directives = [];
+    this.directives.push(
       {
         id: uuidV4(),
         tag: ParseDirective,
         requiredColCount: 1,
       },
       {
         tag: 'divider',
-      },
+      }
+    );
+    if (Theme.showWranglerDatamodelViewer) {
+      this.directives.push(
```

Comment:
We should create a flag called wrangler-data-model and use that flag to guard this feature.  All of our code should be hide behind that flag. wrangler-datamodel-viewer should be removed once we are done.

Reply:
I didn't get what you mean. This flag has been created already, I just used it.

Reply:
Added new `wrangler-data-model` flag.